tests_integration_with_local_chain/markets/seer/test_seer_agent_market.py

```diff
@@ -95,3 +105,60 @@ def test_seer_place_bet_via_pools(
         user_id=test_keys.bet_from_address, outcome=outcome, web3=local_web3
     )
     assert final_outcome_token_balance > 0
+
+
+def test_seer_swap_via_pools(local_web3: Web3, test_keys: APIKeys) -> None:
+    # get open market on Seer with highest liquidity
+    amount_wei = OutcomeToken(1).as_outcome_wei
+    market = SeerAgentMarket.get_markets(
+        limit=1,
+        sort_by=SortBy.HIGHEST_LIQUIDITY,
+        filter_by=FilterBy.OPEN,
+        fetch_categorical_markets=False,
+        fetch_scalar_markets=False,
+    )[0]
+
+    sell_token = market.collateral_token_contract_address_checksummed
+    outcome_idx = 0
+    buy_token = market.wrapped_tokens[outcome_idx]
+
+    # assert there is liquidity to swap
+    assert market.has_liquidity_for_outcome(market.outcomes[outcome_idx])
+
+    # Fund test account
+    collateral_token_contract = to_gnosis_chain_contract(
+        init_collateral_token_contract(
+            market.collateral_token_contract_address_checksummed, local_web3
+        )
+    )
+    auto_deposit_collateral_token(
+        collateral_token_contract=collateral_token_contract,
+        collateral_amount_wei_or_usd=Wei(amount_wei.value),
```

Comment:
```suggestion
        collateral_amount_wei_or_usd=amount_wei,
```

Why not? 😄

Reply:
Ah I see, amount_wei is outcome wei, the name is a bit misleading then 🙈 

```suggestion
        collateral_amount_wei_or_usd=amount_wei.as_wei,
```